src/section/section_peers.rs

```diff
@@ -200,3 +199,32 @@ fn cmp_elder_candidates(
         })
         .then_with(|| lhs.proof.signature.cmp(&rhs.proof.signature))
 }
+
+// Compare candidates for the next elders according to their peer state. The one comparing `Less`
+// wins. `Joined` is preferred over `Relocated` which is preferred over `Left`.
+// NOTE: we only consider `Relocated` peers as elder candidates if we don't have enough `Joined`
+// members to reach `ELDER_SIZE`.
```

Comment:
not sure about this.
I thought `only` Joined shall be considered as candidates.

Reply:
Yes, that's the change. If there is not enough `Joined` members to create `ELDER_SIZE` elders, we fill them up with the current elders that are `Relocated`. That's OK, because such elders are still participating in the section until they are demoted and this will simply delay their relocation further.